src/rust/engine/engine_cffi/src/lib.rs

```diff
@@ -601,17 +606,30 @@ fn workunits_to_py_tuple_value<'a>(workunits: impl Iterator<Item = &'a Workunit>
 pub extern "C" fn poll_session_workunits(
   scheduler_ptr: *mut Scheduler,
   session_ptr: *mut Session,
+  max_log_verbosity_level: u64,
```

Comment:
`externs_set` in this file accepts a `u8` for the log level: should probably try to align with that.

Reply:
It looks like nothing was using this code, so I've removed it.